frontend/src/metabase/admin/permissions/selectors/confirmations.tsx

```diff
@@ -90,6 +90,24 @@ export function getPermissionWarning(
   return null;
 }
 
+export function getTableBlockWarning(
+  dbValue: DataPermissionValue,
+  schemaValue: DataPermissionValue,
+  tableValue?: DataPermissionValue,
+) {
+  if (dbValue === DataPermissionValue.BLOCKED) {
+    return;
+  }
```

Comment:
why is there no DB version of this tooltip?

Reply:
You can see the create queries permissions directly next to the view data permission. I think the thing causing confusion to customers here is that setting one table/schema to blocked is having consequences everywhere else in your db. I'm not sure that this tooltip is the best solution long term, but once we have native query parsing a lot of problems like this go away.